lib.h

```diff
@@ -129,10 +129,13 @@ extern "C" {
 #endif
 
 int tcp_stream(struct options *, struct callbacks *);
+int psp_stream(struct options *, struct callbacks *);
 int udp_stream(struct options *, struct callbacks *);
 int tcp_rr(struct options *, struct callbacks *);
 int udp_rr(struct options *, struct callbacks *);
+int psp_rr(struct options *, struct callbacks *);
 int tcp_crr(struct options *, struct callbacks *);
+int psp_crr(struct options *, struct callbacks *);
```

Comment:
psp_rr and psp_crr are out of scope of this series.

going to merge the pull request, as this is a tiny point -- and that's a good excuse to expedite adding these two too :-)